cypress/support/step_definitions/fireboltCalls.js

```diff
@@ -365,3 +365,125 @@ Given('device is rebooted', () => {
 });
+
+/**
+ * @module fireboltCalls
+ * @function And 3rd party '(.+)' app is dismissed
+ * @description To dismiss the launched app
+ * @param {String} app - app name.
+ * @example
+ * And 3rd party 'firebolt' app is dismissed
+ */
+Given(/3rd party '(.+)' app is dismissed$/, async (appType) => {
+  const appId = Cypress.env(CONSTANTS.RUNTIME).appId;
+  let KeyPressSequence;
+  if (
+    Cypress.env(CONSTANTS.RUNTIME) &&
+    Cypress.env(CONSTANTS.RUNTIME).intent &&
+    Cypress.env(CONSTANTS.RUNTIME).intent.keyPressSequence
+  ) {
+    KeyPressSequence = Cypress.env(CONSTANTS.RUNTIME).intent.keyPressSequence;
+  } else if (
+    Cypress.env('app_metadata') &&
+    Cypress.env('app_metadata')[appId] &&
+    Cypress.env('app_metadata')[appId].defaultKeyPressSequence
+  ) {
+    KeyPressSequence = Cypress.env('app_metadata')[appId].defaultKeyPressSequence;
+  } else if (Cypress.env('app_metadata') && Cypress.env('app_metadata').defaultKeyPressSequence) {
+    KeyPressSequence = Cypress.env('app_metadata').defaultKeyPressSequence;
+  } else {
+    throw new Error('No KeyPressSequence ');
+  }
+  const requestMap = {
+    method: CONSTANTS.REQUEST_OVERRIDE_CALLS.DISMISS,
+    params: KeyPressSequence.dismiss,
+  };
+
+  fireLog.info(
+    `Request sent to platform to dismiss the ${appId} app: ${JSON.stringify(requestMap)}`
+  );
+  cy.sendMessagetoPlatforms(requestMap).then((response) => {
+    fireLog.info(`Response from platform: ${JSON.stringify(response)}`);
+  });
+});
+
+/**
+ * @module fireboltCalls
+ * @function And 3rd party '(.+)' app should be exited
+ * @description To validate that the app is dismissed
+ * @param {String} app - app name.
+ * @example
+ * Then 3rd party 'firebolt' app should be exited
+ */
+Given(/3rd party '(.+)' app should be exited$/, async (app) => {
+  // getAppState validation
+  const appId = Cypress.env(CONSTANTS.CURRENT_APP_ID);
+  const requestMapForGetAppState = {
+    method: CONSTANTS.REQUEST_OVERRIDE_CALLS.GETAPPSTATE,
+    params: appId,
+  };
+  fireLog.info(`Sending request to fetch ${appId} app state: ${JSON.stringify(requestMapForGetAppState)}`);
+  cy.sendMessagetoPlatforms(requestMapForGetAppState)
+    .then((response) => {
+      const responseString = JSON.stringify(response);
+      if (Object.keys(response).length === 0) {
+        fireLog.info(
+          `State validation successful: Current state of ${appId} app is ${responseString} as expected`
+        );
+      } else {
+        fireLog.fail(`${appId} app is not dismissed. Response :${responseString}`);
+      }
+    })
+    .then(() => {
+      // screenShot validation
+      fireLog.info('Started Screenshot validation');
+      const requestMapForScreenShotValidation = {
+        method: CONSTANTS.REQUEST_OVERRIDE_CALLS.SCREENSHOT,
+        params: {
+          validations: [
+            {
+              type: 'image',
+              label: 'home',
+              confidence: 50,
+            },
+          ],
+        },
+      };
```

Comment:
I believe the ask was to create a separate validation object for deeplink and use it here, please check grooming video

Reply:
created deeplink.js file for both screenshot and interactionLogs

Reply:
the feature is dismiss, so we need to create for dismiss

Reply:
updated as Dimiss